### PR TITLE
Add `pyturbo` recipe

### PR DESCRIPTION
**Pull request by adriendelsalle**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pyturbo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  version: {{ version }}
+
+source:
+  - url: https://github.com/twiinit/pyturbo/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 4e777fd460107ad0d1f948e8f230976381c689cc1e6ac2c754670c51698d8615
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (BastianZim) [inline recipes/pyturbo/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```
This is sufficient, the rest will be done by conda-forge.

File: recipes/pyturbo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{% set name = "pyturbo" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/twiinit/pyturbo/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 4e777fd460107ad0d1f948e8f230976381c689cc1e6ac2c754670c51698d8615
```
Comment (BastianZim) [inline recipes/pyturbo/meta.yaml:10]:
```suggestion
  url: https://github.com/twiinit/pyturbo/archive/refs/tags/v{{ version }}.tar.gz
  sha256: 4e777fd460107ad0d1f948e8f230976381c689cc1e6ac2c754670c51698d8615
```
Not really needs, just to standardise it.

File: recipes/pyturbo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    - pip
+  run:
+    - python >=3.8
+    - cosapp >=0.12.0
+    - numpy >=1.14
+    - pyoccad >=0.4.2
+
+test:
+  imports:
+    - pyturbo
```
Comment (BastianZim) [inline recipes/pyturbo/meta.yaml:29]:
```suggestion
    - pyturbo
  commands:
    - pip check
  requires:
    - pip
```

File: recipes/pyturbo/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+about:
+  home: https://github.com/twiinIT/pyturbo
+  summary: Turbomachinery models
+  description: pyTurbo is a library for turbomachinery modeling and simulation based on CoSApp
+  keywords: turbomachinery, CoSApp, twiinit
```
Comment (BastianZim) [inline recipes/pyturbo/meta.yaml:35]: I think this is not supported (anymore).
```suggestion
```

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just a couple of small things, then this is ready to go 🚀

File: recipes/pyturbo/meta.yaml (hunk 1 above)
Comment (adriendelsalle) [reply recipes/pyturbo/meta.yaml:15]: Even the `--no-deps` option has to be removed since I use `pip` to install in a `conda` env?

**Comment by adriendelsalle** [comment]
Thanks @BastianZim and sorry for the late reply.
I'm a bit confused about the `--no-deps` things and left a comment. I'm expecting a wrong behavior if a dep is listed in the `requirements.txt` file and not previously installed from the `host` or `build` sections of the recipe (e.g. the package would also contain the dep).

File: recipes/pyturbo/meta.yaml (hunk 1 above)
Comment (BastianZim) [reply recipes/pyturbo/meta.yaml]: Yes, that is now governed by conda-forge in the build environment. So you don‘t have to set it, it is already set for you. 
If you’re very hesitant you can still set it but there shouldn’t be any difference.

**Review by BastianZim — APPROVED** [review]

File: recipes/pyturbo/meta.yaml (hunk 1 above)
Comment (BastianZim) [reply recipes/pyturbo/meta.yaml]: Just to add: This is, for example, the output of Grayskull which we use for recipes nowadays pretty much all the time.

```yaml
{% set name = "pyturbo" %}
{% set version = "18.7.27" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyturbo-{{ version }}.tar.gz
  sha256: 11aea3102204a16e4f0a05022f1f231ca1c8d40d069b55a64b2eec96f99dc277

build:
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv
  number: 0

requirements:
  host:
    - pip
    - python
  run:
    - numpy
    - opencv
    - python
    - six
    - tensorflow

test:
  imports:
    - pyturbo
  commands:
    - pip check
  requires:
    - pip

about:
  home: https://github.com/aikudexiaohai/pyturbo
  summary: This project contains a variety of different python packages
  license: BSD-2-Clause
  license_file: PLEASE_ADD_LICENSE_FILE

extra:
  recipe-maintainers:
    - ADD_YOUR_GITHUB_ID_HERE
```

**Pull request merged**